addon/snapshot.js

```diff
@@ -41,11 +62,13 @@ export function percySnapshot(name, options) {
   let testingContainer = domCopy.find('#ember-testing');
 
   if (scope) {
-    snapshotHtml = Ember.$('#ember-testing').find(scope).html();
+    snapshotHtml = Ember.$('#ember-testing').find(scope);
```

Comment:
Actually one more thought here — we tried to avoid mutating the state of the real test page at all, so that `percySnapshot` can't be held accountable for any weird failures. Now with this change, this `snapshotHtml` is actually `snapshotRoot` and references the original DOM, which will get affected by the call below.

Can you change this to use `testingContainer.find` instead? I believe that would be equivalent, and would make this not affect the original DOM. Also probably worth it to rename this to `snapshotRoot` for clarity as well, and use `let snapshotHtml =` below.